crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -296,8 +297,12 @@ def proposal?(file)
         raw_data.key?("attributes") && raw_data.key?("deployment")
       end
 
-      def migrate_database(time)
-        Crowbar::Migrate.migrate!
+      def migrate_database(time, migration_level = nil)
+        if migration_level
+          Crowbar::Migrate.migrate_to(migration_level)
```

Comment:
> good point, but is this really possible in the real life? that implies that a user somehow has downgraded his cloud. or where would the user get that backup from?

If we had a db migration over the lifetime of the product, it could happen if the user does a backup with updates, and restores without updates.